### PR TITLE
ci/common/common.sh: shellcheck fixes

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

**Comment by blueyed** [comment]
Only pvs-report failed: https://travis-ci.org/neovim/bot-ci/jobs/242572107#L2350

Is it relevant?

**Comment by blueyed** [comment]
Restarted.

**Comment by blueyed** [comment]
pvs-report fails on master, too: https://travis-ci.org/neovim/bot-ci/jobs/242625902

File: ci/common/common.sh [hunk 1]
```diff
@@ -57,21 +59,21 @@ clone_subtree() {(
...
-  require_environment_variable ${repo} "${BASH_SOURCE[0]}" ${LINENO}
-  require_environment_variable ${branch} "${BASH_SOURCE[0]}" ${LINENO}
+  require_environment_variable "${subtree}" "${BASH_SOURCE[0]}" ${LINENO}
+  require_environment_variable "${dir}" "${BASH_SOURCE[0]}" ${LINENO}
+  require_environment_variable "${repo}" "${BASH_SOURCE[0]}" ${LINENO}
+  require_environment_variable "${branch}" "${BASH_SOURCE[0]}" ${LINENO}
 
   [ -d "${!dir}/.git" ] || git init "${!dir}"
-  cd "${!dir}" 
+  cd "${!dir}" || return
```
Comment (justinmk) [inline ci/common/common.sh]: Should this be an error?

File: ci/common/common.sh [hunk 2]
```diff
@@ -80,15 +82,17 @@ prompt_key_local() {
...
-# $(can_fail_without_private)`.
+# Check whether absence of private (i.e. encrypted) data should fail the build.
+# Echoes 0 in case of pull requests, 1 otherwise.
+# Usage examples:
+# - `exit $(can_fail_without_private)`
+# - `return $(can_fail_without_private)`
 can_fail_without_private() {
-  if test "$TRAVIS_EVENT_TYPE" = pull_request ; then
+  if [ "$TRAVIS_EVENT_TYPE" = pull_request ]; then
     echo 0
```
Comment (justinmk) [inline ci/common/common.sh]: What was shellcheck complaint here ?

File: ci/common/common.sh [hunk 3]
```diff
@@ -107,32 +111,32 @@ commit_subtree() {(
...
-  require_environment_variable ${dir} "${BASH_SOURCE[0]}" ${LINENO}
-  require_environment_variable ${repo} "${BASH_SOURCE[0]}" ${LINENO}
-  require_environment_variable ${branch} "${BASH_SOURCE[0]}" ${LINENO}
+  require_environment_variable "${subtree}" "${BASH_SOURCE[0]}" ${LINENO}
+  require_environment_variable "${dir}" "${BASH_SOURCE[0]}" ${LINENO}
+  require_environment_variable "${repo}" "${BASH_SOURCE[0]}" ${LINENO}
+  require_environment_variable "${branch}" "${BASH_SOURCE[0]}" ${LINENO}
 
-  cd "${!dir}"
+  cd "${!dir}" || return
```
Comment (justinmk) [inline ci/common/common.sh]: Error?

File: ci/common/common.sh (hunk 2 above)
Comment (blueyed) [inline ci/common/common.sh]: I think that was just me, for consistency.

File: ci/common/common.sh (hunk 1 above)
Comment (blueyed) [inline ci/common/common.sh]: It will return non-zero.
Is that enough?

**Pull request merged**